package.json

```diff
@@ -29,6 +29,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.7.0",
+    "@testing-library/react-hooks": "^3.4.2",
```

Comment:
moving these packages to `foreman-js`

Reply:
https://github.com/theforeman/foreman-js/pull/237

Reply:
can you please update the package.json to require the correct version of `@theforeman/test` which includes these?